controller/space_template.go

```diff
@@ -0,0 +1,26 @@
+package controller
+
+import (
+	"github.com/fabric8-services/fabric8-wit/app"
+	"github.com/goadesign/goa"
+)
+
+// SpaceTemplateController implements the space_template resource.
+type SpaceTemplateController struct {
+	*goa.Controller
+}
+
+// NewSpaceTemplateController creates a space_template controller.
+func NewSpaceTemplateController(service *goa.Service) *SpaceTemplateController {
+	return &SpaceTemplateController{Controller: service.NewController("SpaceTemplateController")}
+}
+
+// Show runs the show action.
+func (c *SpaceTemplateController) Show(ctx *app.ShowSpaceTemplateContext) error {
+	// SpaceTemplateController_Show: start_implement
+
+	// Put your logic here
```

Comment:
Yes, that controller will be empty till we have space_template in place.
We need this because we build the endpoint `/api/spacetemplates/template_ID/workitemtypegroups`
here, in the design we use spacetemplate as a parent to typegroups - https://github.com/fabric8-services/fabric8-wit/pull/1541/files#diff-1b26074d4e13d2ec3ec3fe3ac0a48507R56